launch_testing_ament_cmake/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.5)
 
-project(launch_testing_ament_cmake NONE)
+project(launch_testing_ament_cmake)
```

Comment:
This is necessary for `FindPythonLibs` to determine the Python libraries and include dirs.